src/sbvr-api/abstract-sql.ts

```diff
@@ -232,3 +237,37 @@ export const isRuleAffected = (
 	}
 	return checkModifiedFields(rule.ruleReferencedFields, modifiedFields);
 };
+
+export const namespaceRelationships = (
+	relationships: Relationship,
+	alias: string,
+	opts: {
+		keyFilter?: (key: string) => boolean;
+		deleteOriginal: boolean;
+	},
+): void => {
+	for (const [key, relationship] of Object.entries(
+		relationships as RelationshipInternalNode,
+	)) {
+		if (key === '$') {
+			continue;
+		}
+
+		let mapping = (relationship as RelationshipLeafNode).$;
+		if (mapping != null && mapping.length === 2) {
+			if (opts?.keyFilter?.(key) !== false) {
```

Comment:
Took me a bit to realize that this means v, but it might be b/c it's Monday :)
```suggestion
			if (opts.keyFilter == null && opts.keyFilter(key)) {
```

Reply:
The alternative would actually be `if (opts.keyFilter == null || opts.keyFilter(key) !== false) {` btw